### PR TITLE
Challenge submission: VGG19 pretrained model

### PR DESCRIPTION
**Pull request by coreen**
Passed challenge requirements of >= 20% accuracy for new flower images

### CODE REVIEW

**Comment by coreen** [comment]
Reflection on project:
* Good overall coverage of deep learning through PyTorch framework (CarND uses Tensorflow with Keras)
* CNN, RNN, and GAN exercises accompanying course cover recent publications
* Additional links at beginning of lessons are great for digging deeper into particular types of networks
